### PR TITLE
Astyle fixes

### PR DESCRIPTION
**Pull request by bazsi**
This patches contains a number of style fixes that a newer version of astyle finds. They make sense,
and I was motivated to do this as the current astyle is insisting on fixing an issue in an ugly way.

I am trying to find a way to run the newer version of astyle in our automated builds.

### CODE REVIEW

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Comment by kira-syslogng** [comment]
Build SUCCESS

File: .travis.yml [hunk 1]
```diff
@@ -114,10 +116,10 @@ matrix:
     - env: B=style-check
       dist: trusty
       sudo: required
-      before_script:
-      - astyle --version
+      before_install:
+      - docker pull balabit/syslog-ng-image-bionic
       script:
-      - scripts/style-checker.sh format
+      - ./dbld/rules make-style-format
```
Comment (Kokan) [inline .travis.yml]: Running the same script inside dbld image would be fine - even better than calling any configure & make. I could imagine something like this: `dbld/rules shell scripts/style-checker.sh format` (without knowing much about dbld).

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Comment by gaborznagy** [comment]
Don't forget to merge the following PR either:
https://github.com/balabit/syslog-ng-gitbook/pull/64

**Pull request merged**